### ISSUE TITLE
After training for a period of time, the loss cannot be lost

### ISSUE THREAD

**Issue by TenderSu** [EdoardoBotta/RQ-VAE-Recommender#21]
Hi Botta, Thank you for your work，but when I reproduced your work, I encountered some problems. 
I pulled commit b9d6ed7 in which I saw you used the title embedding of the movie as an input feature, when I follow your code and use the sentence t5 model as embedding model, i found the training loss  will only decrease in the initial stage. When the training reaches about 2000 iterations, the loss will remain around 5. What causes this?

**Comment by EdoardoBotta** [comment]
This was caused by some experiment I was performing with the CategoricalReconstructionLoss, trying to integrate the movie genres features. Ultimately, I decided not to use the genre features at all and reconstruct the title embedding only, as this approach is more similar to the original paper. I would recommend to pull from master and use the latest version of the train_rqvae.py as I have significantly updated it in the last weeks.

**Comment by yujin39** [comment]
I have successfully ran the train_rqvae.py, but I am encountering the same issue of staggering loss when running train_decoder.py.

![image](https://github.com/user-attachments/assets/8c3741ff-ebef-4b70-a7a8-a4e8a0215171)
